common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -29,7 +29,7 @@
 /**
  * The internal representation of interval type.
  */
-public final class CalendarInterval implements Serializable, Comparable<CalendarInterval> {
+public final class CalendarInterval implements Serializable {
```

Comment:
there is a `MAX_VALUE` and `MIN_VALUE` in this file, shall we remove them as well?

Reply:
yes, remove them too